docs/CHANGELOG.md

```diff
@@ -41,7 +41,9 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 - Caret for editable text box is positioned correctly on first click. ([#2151])
 - Editable text is correctly blurred / unfocused when clicking outside its bounds. ([#2091], [#2151])
 - `amethyst_test` crate features now map 1-1 to `amethyst` features. ([#2153])
+- Corrected an issue where fixed updates were tied to time scale. ([#2254])
```

Comment:
Can you move this to the unreleased section as defined in https://keepachangelog.com/en/1.0.0/ ?